client/my-sites/site-logs/components/site-logs-table/site-logs-expanded-content.tsx

```diff
@@ -0,0 +1,23 @@
+import { SiteLogsData } from 'calypso/data/hosting/use-site-logs-query';
+import './style.scss';
+
+interface Props {
+	log: SiteLogsData[ 'logs' ][ 0 ];
+}
+
+export default function SiteLogsExpandedContent( { log }: Props ) {
+	return (
+		<div className="site-logs-table__expanded-content">
+			{ Object.keys( log ).map( ( key ) => (
+				<tr key={ `expanded-row-${ key }` }>
```

Comment:
Just a note that the React keys don't have to be unique across multiple components or anything

```suggestion
				<tr key={ key }>
```